pkg/pool-weighted/contracts/WeightedPool.sol

```diff
@@ -15,21 +15,14 @@
 pragma solidity ^0.7.0;
 pragma experimental ABIEncoderV2;
 
-import "@balancer-labs/v2-solidity-utils/contracts/math/FixedPoint.sol";
-import "@balancer-labs/v2-solidity-utils/contracts/helpers/InputHelpers.sol";
-
-import "@balancer-labs/v2-pool-utils/contracts/BaseMinimalSwapInfoPool.sol";
-
-import "./WeightedMath.sol";
-import "./WeightedPoolUserDataHelpers.sol";
+import "./BaseWeightedPool.sol";
 
 // This contract relies on tons of immutable state variables to perform efficient lookup, without resorting to storage
 // reads. Because immutable arrays are not supported, we instead declare a fixed set of state variables plus total
 // count, resulting in a large number of state variables.
 
```

Comment:
```suggestion
/**
 * @dev Basic Weighted Pool with immutable weights.
 */
```

The state variables comment was copy-pasted by mistake from `BasePool`: it is meant to be an explanation of why a certain solhint check is disabled there.